scripts/dummy_script.py

```diff
@@ -0,0 +1,397 @@
+from decimal import Decimal
+
+import pandas as pd
+
+from hummingbot.core.data_type.common import OrderType, TradeType
+from hummingbot.core.data_type.order_candidate import OrderCandidate
+from hummingbot.core.event.events import OrderFilledEvent
+from hummingbot.strategy.script_strategy_base import ScriptStrategyBase
+
+
+class DummyScript(ScriptStrategyBase):
+    order_amount = Decimal(0.5)   # This can be adjusted
+    order_refresh_time = 10   # This can be adjusted
+    create_timestamp = 0
+
+    base_asset = "BTC"   # This can be adjusted
+    quote_asset = "USDT"   # This can be adjusted
+    trading_pair = f"{base_asset}-{quote_asset}"
+
+    high_liquidity_exchange = "binance_paper_trade"   # This can be adjusted
+    low_liquidity_exchange = "gate_io_paper_trade"   # This can be adjusted
+    markets = {high_liquidity_exchange: {trading_pair},
+               low_liquidity_exchange: {trading_pair}
+               }
+
+    # These 2 attributes are used to keep track of the amount of open orders on the low liquidity exchange
+    buy_orders_on_low_liquidity_exchange = 0
+    sell_orders_on_low_liquidity_exchange = 0
+
+    # These 2 attributes have a default value of False. When there is an order filled on the low liquidity exchange
+    # it will be set to True so the bot knows it should place a hedge order on the high liquidity exchange
+    buy_order_completed = False
+    sell_order_completed = False
+
+    # this will be used to calculate the price for the hedge order on the high liquidity exchange:
+    spread_bps = 5
+    min_spread_bps = 0
+
+    def high_liquidity_connector(self):
+        return self.connectors[self.high_liquidity_exchange]
+
+    def on_tick(self):
+        """Check if orders need to be refreshed. If yes we need to cancel all orders, calculate new price and amount and place
+        new maker orders on the low liquidity exchange. If no than we need to check that we have maker orders on both
+        bid and ask side. We also need to check if an order got filled. If yes than we need the bot to place a hedge
+        order on the high liq ex. We also want the bot to check the balances and send a notification if an available
+        balance runs low."""
+        # Low liquidity exchange side
+        # Do we need to refresh orders?
+        if self.create_timestamp <= self.current_timestamp:
+            self.cancel_all_orders()
+            proposal = self.create_proposal_based_on_high_liquidity_exchange_price()
+            adjusted_proposal = self.adjust_proposal_to_budget(proposal)
+            self.place_orders(adjusted_proposal, self.low_liquidity_exchange)
+            # Update timestamp
+            self.create_timestamp = self.order_refresh_time + self.current_timestamp
+
+        # Are there less than 1 buy or sell order on the exchange?
+        elif self.sell_orders_on_low_liquidity_exchange < 1 or self.buy_orders_on_low_liquidity_exchange < 1:
```

Comment:
After this condition is met, the logic is the same of the previous block, so instead of adding this two blocks you can just add:
if self.create_timestamp <= self.current_timestamp or (self.sell_orders_on_low_liquidity_exchange < 1 or self.buy_orders_on_low_liquidity_exchange < 1):
and then the code.